### PR TITLE
Import community docs

### PR DESCRIPTION
**Pull request by dholbach**
This is a bit of a wild draft to get us started with #50 - let's clean it up together.

### CODE REVIEW

File: content/en/community.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+---
```
Comment (hiddeco) [inline content/en/community.md:1]: Shouldn't this file be included in the `.gitignore` so that the latest version is pulled during build?

Comment (dholbach) [reply content/en/community.md:1]: Hum... not sure what to make of https://github.com/kubernetes-sigs/contributor-site/commit/69818fa3a79c39812044c8d095aa2eb2f3d4329f - I guess we'd need to do this in a similar fashion?

Comment (hiddeco) [reply content/en/community.md:1]: Means that it should be excluded, see the combination of:

https://github.com/kubernetes-sigs/contributor-site/blob/69818fa3a79c39812044c8d095aa2eb2f3d4329f/content/en/docs/.gitignore + https://github.com/kubernetes-sigs/contributor-site/blob/69818fa3a79c39812044c8d095aa2eb2f3d4329f/external-sources/kubernetes/community#L17

**Comment by dholbach** [comment]
Some questions:
- Do we need to hook up the `gen-content` target in the Makefile somewhere else?
- Do we want to note down somewhere where we copied `hack/gen-content.sh` from?
- I guess we need to rework improve styling a little in `layouts/_default/single.html` - like spacing at top and bottom?

File: Makefile [hunk 2]
```diff
@@ -1,16 +1,30 @@
...
+                                        fcntl.fcntl(sys.stdout, fcntl.F_SETFL, flags&~os.O_NONBLOCK);"
+
+gen-content: ## Generates content from external sources.
+	hack/gen-content.sh
+
+container-gen-content: ## Generates content from external sources within a container (equiv to gen-content).
+	$(CONTAINER_RUN) $(CONTAINER_IMAGE) hack/gen-content.sh
+
 serve:
 	hugo server \
```
Comment (hiddeco) [inline Makefile:14]:
```suggestion
serve:
	hack/gen-content.sh
	hugo server \
```

Otherwise the pages will not be present after cloning.

**Review by hiddeco — APPROVED** [review]
Thank you Daniel :grapes:

**Comment by dholbach** [comment]
Thanks a lot @hiddeco :strawberry:

**Pull request merged**